.gitignore

```diff
@@ -1,3 +1,5 @@
 dist
 node_modules
 .idea
+src/**/*.js
```

Comment:
Why would there be any JS files in these directories?

Reply:
not sure, I think the .gitignore came from appeals repo.

Reply:
I added that, I'll remove it.